src/query/server/query.go

```diff
@@ -504,6 +504,13 @@ func Run(runOpts RunOptions) RunResult {
 			}
 		}()
 
+		logger.Info("configuring downsampler to use with aggregated namespaces",
+			zap.Int("numAggregatedClusterNamespaces", opts.Namespaces().NumAggregatedClusterNamespaces()))
+		err = clusterNamespacesWatcher.Update(opts.Namespaces())
+		if err != nil {
+			logger.Error("error when updating namespaces", zap.Error(err))
+		}
```

Comment:
Hm I think this should be a fatal error yeah? Otherwise coordinator will come up and no writes will succeed. Better to crash the process due to bad config here so you don't rollout unknowingly. (nit: can also scope the err to if clause)
```suggestion
		if err := clusterNamespacesWatcher.Update(opts.Namespaces()); err != nil {
			logger.Fatal("error when updating namespaces", zap.Error(err))
		}
```

Reply:
yeah, my mistake. Will fix.